Tools/autotest/autotest.py

```diff
@@ -623,6 +624,11 @@ def run_tests(steps):
                            help='do not configure before building',
                            dest="no_configure")
     group_build.add_option("-j", default=None, type='int', help='build CPUs')
+    group_build.add_option("--use-clang",
```

Comment:
I'd prefer to make this symmetric with `sim_vehicle.py` with a waf-configure-args thing:

https://github.com/ArduPilot/ardupilot/pull/9401/commits/6a2f21cd12c3b7adab367c5c5ce3cbdde3508342